dashboard/index.js

```diff
@@ -28,9 +43,13 @@ const launchBroker = (port) => {
         brokerProcess.unref();
         brokerProcess.disconnect();
 
-        // clean exit the process
-        process.exit(0);
+        if (!isTest) {
+            // clean exit the process
+            process.exit(0);
```

Comment:
Instead of adding a check for `isTest`, you can stub the `process.exit` and assert that it was called.